CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+All notable changes to the Solar Forecast Arbiter Dashboard will be documented
```

Comment:
This implies to me that this just tracks the Dashboard, but it's really anything that a dashboard user would encounter.

We should also encourage users to read the [`solarforecastarbiter-core` what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.